cmd/gendeepcopy/deep_copy.go

```diff
@@ -125,8 +125,9 @@ func main() {
 			generator.OverwritePackage(vals[0], vals[1])
 		}
 	}
+
 	var schemeVersion string
-	if version == "" {
+	if *groupVersion == "" || *groupVersion == "/" {
```

Comment:
actually, with the changes at https://github.com/kubernetes/kubernetes/pull/17432/files#diff-ded85789c18cc8c3fc7b4038b75423a8R57, who is producing group version strings of "/"?